tools/clang/lib/CodeGen/CodeGenTypes.cpp

```diff
@@ -14,21 +14,23 @@
 #include "CodeGenTypes.h"
 #include "CGCXXABI.h"
 #include "CGCall.h"
+#include "CGHLSLRuntime.h" // HLSL Change
 #include "CGOpenCLRuntime.h"
 #include "CGRecordLayout.h"
+#include "CodeGenModule.h" // HLSL Change
 #include "TargetInfo.h"
+#include "dxc/DXIL/DxilUtil.h"
 #include "clang/AST/ASTContext.h"
 #include "clang/AST/DeclCXX.h"
-#include "clang/AST/DeclTemplate.h"
 #include "clang/AST/DeclObjC.h"
+#include "clang/AST/DeclTemplate.h"
```

Comment:
I know this isn't really a change so much as clang-format, but marking it makes comparing to Clang 3.7 easier.

```suggestion
#include "clang/AST/DeclTemplate.h" // HLSL Change - clang-format
```